api_generator/Cargo.toml

```diff
@@ -9,9 +9,9 @@ edition = "2018"
 license = "Apache-2.0"
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
This change seems to be not related (reasonable simplification of error handling), could we move it off to separate pull request? ty

Reply:
Fair enough, have split those changes out